docs/blog/2016-9-28.md

```diff
@@ -0,0 +1,15 @@
+# New Release
+We continue our montly release cycle with our [September release](https://github.com/vmware/docker-volume-vsphere/releases/tag/0.7)
+
+Thanks to [Bruno Moura's](https://github.com/brunotm) excellent [work](https://github.com/vmware/docker-volume-vsphere/pull/597), Docker volume plugin for vSphere now supports selection of file system format when creating a new docker volume.
+
+Ability to set a volume [read-only](https://github.com/vmware/docker-volume-vsphere/issues/274) is now supported by the plugin. The [admin cli](https://vmware.github.io/docker-volume-vsphere/user-guide/admin-cli/#set) allows the admin to set a volume to be read only.
+
+# Coming up
+
+One of the major features in the pipeline is multi-tenancy and vCenter integration. More about that soon..
+
+# External Blog entries
+
+- Cormac Hogan - [Docker Volume Driver for vSphere using policies on VSAN](http://cormachogan.com/2016/09/26/docker-volume-driver-vsphere-using-policies-vsan-short-video/)
+- William Lam - [Getting Started with Tech Preview of Docker Volume Driver for vSphere - updated](http://www.virtuallyghetto.com/2016/05/getting-started-with-tech-preview-of-docker-volume-driver-for-vsphere.html)
```

Comment:
There are more;
http://cormachogan.com/2016/06/01/docker-volume-driver-vsphere/
http://cormachogan.com/2016/06/03/docker-volume-driver-vsphere-short-video/
http://cormachogan.com/2016/06/09/docker-volume-driver-vsphere-virtual-san-vsan/
http://cormachogan.com/2016/07/29/using-vsphere-docker-volume-driver-run-project-harbor-vsan/
(New) http://cormachogan.com/2016/10/06/nice-enhancements-docker-volume-driver-vsphere-v0-7/

https://blogs.vmware.com/virtualblocks/2016/06/20/vsphere-docker-volume-driver-brings-benefits-of-vsphere-storage-to-containers/

It may makes sense to categorize them in Intro, Advanced etc